Library/Homebrew/utils/bottles.rb

```diff
@@ -75,6 +75,36 @@ def formula_contents(bottle_file,
         contents
       end
 
+      def merge_bottle_spec(old_bottle_spec, new_bottle_hash)
```

Comment:
Do you see this being used anywhere other than `dev-cmd/bottle.rb`? If not: I think it'd be nicer to leave it there.